### DISCUSSION TITLE
Where to download page-flip.browser.js file?

### DISCUSSION THREAD

**Discussion by amanking** [Nodlik/StPageFlip#33] [Q&A]
I do not want to use NPM but directly use vanilla JS on a website using a script tag. The instructions mention a page-flip.browser.js file but I cannot find it in this repository. There's a reference to download the bundle from Github but I couldn't find it. Please advise. Thanks.

**Comment by kkk0101** [comment]
https://cdn.jsdelivr.net/npm/page-flip/dist/js/page-flip.browser.min.js
